core/src/main/scala/org/apache/spark/api/python/PythonRunner.scala

```diff
@@ -354,7 +355,8 @@ private[spark] abstract class BasePythonRunner[IN, OUT](
     extends Thread(s"Worker Monitor for $pythonExec") {
 
     /** How long to wait before killing the python worker if a task cannot be interrupted. */
-    private val taskKillTimeout = env.conf.getTimeAsMs("spark.python.task.killTimeout", "2s")
+    private val taskKillTimeoutMs = env.conf.getTimeAsSeconds("spark.python.task.killTimeout",
```

Comment:
The conf is not documented, but I think it's designed to accept values like 1.5s
cc @zsxwing